spec/tests/helpers/perf_run_helper_spec.rb

```diff
@@ -439,32 +439,68 @@ class PerfRunHelperClass
     end
   end
   describe "#copy_system_logs" do
```

Comment:
```suggestion
  
  describe "#copy_system_logs" do
```

Reply:
This suggestion reduced the indentation of the describe line by one space.  Please add the space back to the line.